src/Services/Logger.ts

```diff
@@ -1,17 +1,39 @@
 import winston from "winston";
+import ElasticsearchTransport, {LogData} from "winston-elasticsearch"
+import {ClientOptions} from "@elastic/elasticsearch"
+
+import config from "../Config/Config";
 
 const logger = winston.createLogger({
-    format: winston.format.combine(
-        winston.format.colorize(),
-        winston.format.timestamp(),
-        winston.format.json(),
-    ),
     level: "info",
+    transports: new winston.transports.Console({
+        format: winston.format.combine(
+            winston.format.timestamp({
+                format: 'YYYY-MM-DD HH:mm:ss'
+            }),
+            winston.format.printf(info => `${info.timestamp} | ${info.level} | ${info.message}`),
+        ),
+    }),
 });
 
-logger.add(new winston.transports.Console({
-    format: winston.format.simple(),
-}));
+const esTransportOpts = {
+    level: 'info',
+    transformer: (logData: LogData) => {
+        return {
+            "@timestamp": (new Date()).toLocaleString(),
+            severity: logData.level,
+            message: logData.message
+        }
+    },
+    clientOpts: {
+        node: `${config.elasticsearch.url}:${config.elasticsearch.port}`,
+        log: `${config.elasticsearch.level}`
+    } as ClientOptions,
+};
+
+if (config.env == "prod") {
```

Comment:
@nmlinaric I've set NODE_ENV=production not "prod", also put here `if(config.logging.es.enabled)`